src/redprl/judgment.fun

```diff
@@ -12,7 +12,7 @@ struct
 
   val subst = S.map o Tm.substMetaenv
   val eq = S.eq
-  val toString = S.toString TermPrinter.toString
+  val toString = FppRenderPlainText.toString o FinalPrinter.execPP o S.pretty TermPrinter.ppTerm
```

Comment:
@favonia I consider it a bug if we ever use that from within RedPRL; the `toString` stuff is required by the LCF library, in order to make it possible to debug.